dashboard/app/controllers/api/v1/teacher_feedbacks_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::V1::TeacherFeedbacksController < Api::V1::JsonApiController
+  load_and_authorize_resource
+
+  # Use student_id, level_id, and teacher_id to lookup the feedback for a student on a particular level and provide the
+  # most recent feedback left by that teacher
+  def show_feedback_from_teacher
+    @feedback = TeacherFeedback.where(student_id: params[:student_id], level_id: params[:level_id], teacher_id: params[:teacher_id]).last
```

Comment:
nit: split this up for readability:
```ruby
@feedback = TeacherFeedback.where(
  student_id: params[:student_id],
  level_id: params[:level_id],
  teacher_id: params[:teacher_id]
)...
```

Why `last`? Note there is no explicit order here. Do we expect an order? If we want this to be a certain one, like the most recent, then do something like
```ruby
.order('id DESC').first
```
otherwise just use the first.